sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -190,6 +190,19 @@ class PreProcessingDependency(Base):
     relationship = Column(DEPENDENCY_RELATIONSHIPS, nullable=False)
     direction = Column(DEPENDENCY_DIRECTIONS, nullable=False)
 
+    def reverse_direction(self):
```

Comment:
I think it's ok to do this for this SIT but it won't be true after that when we add in inter-instrument dependency. Can you note that somewhere as reminder?

Reply:
I thought of an idea for this case. I will share that after SDWG meeting.

Reply:
But I don't mean to not merge this. You can merge this because I will need to make same changes for other instruments. I can do that after this PR is merged.